.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
 script:
   - npm run lint
   - npm test
+  - npm coverage
```

Comment:
I would have caught this bug if you had given me the chance to review it 😁

Reply:
Haha I didn't want coverage to drop with the introduction of another PR, it was an emergency 😄